btp.lua

```diff
@@ -0,0 +1,486 @@
+                                   firm_type_string, nil, nil)
+local user_memo_len = ProtoField.uint8("btp.user_memo_len", "User Memo Length",
+                                       base.DEC, nil, nil, nil)
+local user_memo = ProtoField.string("btp.user_memo", "User Memo", base.ASCII,
+                                    nil)
+local modify_id = ProtoField.uint64("btp.modify_id", "Modify ID", base.DEC, nil,
+                                    nil, nil)
+local old_price = ProtoField.int32("btp.old_price", "Old Price", base.DEC, nil,
+                                   nil, nil)
+local old_quantity = ProtoField.uint32("btp.old_quantity", "Old Quantity",
+                                       base.DEC, nil, nil, nil)
+local filled_quantity = ProtoField.uint32("btp.filled_quantity",
+                                          "Filled Quantity", base.DEC, nil, nil,
+                                          nil)
+local liquidity = ProtoField.uint8("btp.liquidity", "Liquidity", base.HEX,
+                                   liquidity_string, nil, nil)
+local close_reason = ProtoField.uint8("btp.close_reason", "Close Reason",
+                                      base.HEX, close_reason_string, nil, nil,
+                                      nil)
+local order_entry_reject_reason = ProtoField.uint8(
+                                      "btp.order_entry.reject_reason",
+                                      "Reject Reason", base.HEX,
+                                      order_entry_reject_reason_string, nil, nil)
+local time_in_force = ProtoField.uint8("btp.time_in_force", "Time in Force",
+                                       base.HEX, time_in_force_string, nil, nil)
+local order_entry_message_type = ProtoField.uint8(
+                                     "btp.order_entry.message_type",
+                                     "Message Type", base.HEX,
+                                     order_entry_message_type_string, nil, nil)
+local drop_copy_message_type = ProtoField.uint8("btp.drop_copy.message_type",
+                                                "Message Type", base.HEX,
+                                                drop_copy_message_type_string,
+                                                nil, nil)
+
+local price_feed_message_type = ProtoField.uint8("btp.price_feed.message_type",
+                                                 "Message Type", base.HEX,
+                                                 price_feed_message_type_string,
+                                                 nil, nil)
+local login_message_type = ProtoField.uint8("btp.login.message_type",
+                                            "Message Type", base.HEX,
+                                            login_message_type_string, nil, nil)
+
+local function dissect_order_entry(buffer, pinfo, tree)
+    tree:add_le(order_entry_message_type, buffer:range(0, 1))
+    local mt = buffer:range(0, 1):string()
```

Comment:
You get `buffer:range(0, 1)` twice, might as well put that in a variable, like `mt_bytes`?

Reply:
Yep, fixed